scrapers/javlibrary.yml

```diff
@@ -2,7 +2,10 @@ name: JavLibrary
 sceneByFragment:
   action: scrapeXPath
   queryURL: https://www.javlibrary.com/en/vl_searchbyid.php?keyword={title}
-  #Note: Use the title of your scene. So if your filename is SSNI-000.mp4, remove the extension, save, then use the scraper
+  # Note: The scraper uses the title from your scene to search for the movie id.
+  # Make sure that the title is saved in the correct form e.g. SSNI-000 ( no extensions or other extra words/characters should be present) before you use the scrape with function.
+  # If javlibrary doesn't work, you can replace javlibrary with any mirrors in the queryURL.
+  # If the movie have a Blu-Ray version, the scraper will set the URL of the DVD version. So you need to use the sceneScraper by URL after.
```

Comment:
typo
If the movie **has** a Blu-Ray version,  the scraper will set the URL of the DVD version.

If i understand correctly what you mean though  this could be rephrased to


`The scraper matches the DVD version of a title. If you want the Blu-Ray version you'll need to add the correct URL manually and scrape using that afterwards.`

From a quick look it seems to me that most bluray versions have an extra tag `Blu-ray` added and the `Risky Mosaic` one removed ( uncensored version ? ).

Reply:
### Exemple: 
If your title is SSNI-888, the scraper will get this page:
`https://www.javlibrary.com/en/vl_searchbyid.php?keyword=SSNI-888`
There is 2 results, one is the Blu-Ray and DVD.
So by default the scraper will take the URL of the movie without "Blu-Ray" in title. (Then you will use the icon near URL to scrape it)